NTumbleBit/ClassicTumbler/Client/StateMachinesExecutor.cs

```diff
@@ -12,6 +12,8 @@ namespace NTumbleBit.ClassicTumbler.Client
 {
     public class StateMachinesExecutor : TumblerServiceBase
     {
+        protected HashSet<CycleParameters> ManagedCycles = new HashSet<CycleParameters>();
```

Comment:
Will we need to populate this on startup, or will it only affect cycles being newly created?

Reply:
This apply only to new cycles, but to be hones I didn't consider a case when we start the node up and load cycles from repository and continue tumbling. Does that ever happen?